Extensions/Leaderboards/leaderboardstools.ts

```diff
@@ -279,30 +280,42 @@ namespace gdjs {
         let scoreSavingState: ScoreSavingState;
         if (_scoreSavingStateByLeaderboard[leaderboardId]) {
           scoreSavingState = _scoreSavingStateByLeaderboard[leaderboardId];
+          let shouldStartSaving = true;
           if (
+            shouldStartSaving &&
```

Comment:
this is probably not necessary, as you just created the value to true

Reply:
In case someone starts moving those 3 ifs around, they won't get unexpected errors